includes/templates/bootstrap/templates/tpl_account_history_info_default.php

```diff
@@ -177,9 +177,9 @@
                             </div>
                         </div>
 <?php
-} else { // temporary just remove these 4 lines
+} else {
 ?>
-                        <div>WARNING: Missing Shipping Information</div>
+                        <div><?php echo TEXT_MISSING_SHIPPING_INFO; ?></div>
```

Comment:
The associated constant wasn't defined until after Zen Cart 1.5.8a and the template supports the zc157 family as well.

That said, the language files in the `languages/english/extra_definitions` sub-directory (both the zc158 and legacy versions) will require this constant to be added.

Reply:
Will look at and update.

Reply:
Added missing language file elements.
Put version check in as over entry in languages/english/account_history_info.php

If you want to leave for 3.7.0 that fine.